chart/compass/charts/kyma-environment-broker/values.yaml

```diff
@@ -48,8 +48,10 @@ additionalRuntimeComponents:
     namespace: "kyma-system"
   - name: "compass-runtime-agent"
     namespace: "compass-system"
+  - name: "knative-eventing-kafka"
+    namespace: "knative-eventing"
 
-kymaVersion: "1.10.0"
+kymaVersion: "master-99c5dca2"
```

Comment:
I'm not sure we should propose this SHA change. :p

Reply:
It is used to set the `APP_KYMA_VERSION` environment variable in the KEB deployment. We need to update it to have the resources for knative-eventing-kafka inside the installer image. We added the component files in this [PR](https://github.com/kyma-project/kyma/pull/7486). So we need a newer Kyma version than 1.10. We could also use 1.11 as soon as it is out.